tensorflow_recommenders_addons/dynamic_embedding/core/lib/nvhash/nv_hashtable.cuh

```diff
@@ -373,13 +372,15 @@ public:
         set_kernel<<<grid_size, BLOCK_SIZE_, 0, stream>>>(table_, d_keys, (const ValType*)d_vals, len);
     }
 
-    /*void accum(const KeyType* d_keys, const ValType* d_vals, size_t len, cudaStream_t stream){
+    void accum(const KeyType* d_keys, const BaseValType* d_vals_or_deltas, const bool* d_exists, size_t len,
+               cudaStream_t stream) {
         if (len == 0) {
             return;
         }
         const int grid_size = (len - 1) / BLOCK_SIZE_ + 1;
```

Comment:
Here it assigns one thread for each KV's accumulation. Maybe need to raise an error when `len` becomes too large, while in most cases of GPU hardware it will not happen.

Reply:
The grid size could be "Max dimension size of a grid size    (x,y,z): (2147483647, 65535, 65535)",  so `grid_size ` would never be larger than up limit.
Refer to https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#features-and-technical-specifications

Reply:
Maybe leave a comment here.

Reply:
Considering the possible up limit of `len` will be equal to `256*(2^31-1-1)+1=5.49^11`, the GPU device memory will be reached firstly, we will change it in this PR for uniform code style.